pom.xml

```diff
@@ -4,13 +4,13 @@
   <parent>
     <groupId>org.nuxeo</groupId>
     <artifactId>nuxeo-parent</artifactId>
-    <version>2023.1</version>
+    <version>2025.0.31</version>
```

Comment:
Maybe better to leverage `2025.0-SNAPSHOT` until we release 2025.

Reply:
yes indeed